modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,60 +35,59 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes the following new features and capabilities:
 
-- {proglang} and the JavaScript library now support floating point numbers (Float).
-- When using the Candid UI, you can view return values in text, UI, and JSON formats.
-+
-You can switch between the formats for representing values by clicking the results bar.
-- Internal updates that are not yet user-visible to support future features.
+=== SDK
 
-== Breaking change
-The {release} release includes the following changes that might require updates to existing programs:
-
-- The name of the built-in {proglang} library has changed from `stdlib` to `base`.
-- All of the module names in the `base` library start with an upper case letter.
-- The `principalId` module has been changed to `Principal`.
+- You can now look up a canister identifier using the command `dfx canister id <canister_name>`.
+- The `--check` flag can be used with the `dfx build` command to check whether a canister will build before creating or building the canister.
+- Both canister name and identifiers are printed upon canister creation.
+- Canister manifest now stores multiple canister IDs associated with local and remote networks as well as network mapping.
 
-=== Modifying existing programs
+=== Tungsten (Developer Network)
 
-If you have programs that rely on modules in the `stdlib` library, you must change the `import` statement to use the new `base` library name.
-For example, you might have previously imported the Array module using the following statement:
+- HTTP authorization and credentials management for onboarded Tungsten users.
+- `Tungsten` added as a provider in `dfx.json`
+- You can use the `dfx ping` command to access and toggle between local and Tungsten networks.
+- The `--network <network>` flag can be used to build and install canisters to the specified provider.
+- Canister ID formatting for accessing Tungsten-deployed apps in the browser.
 
-[source,motoko]
-----
-import Array "mo:stdlib/array";
-----
+=== Motoko
 
-With this release, you would change the statement like this:
+- Stable variable support
+- Improved type checking of literals as payload for variant tags. The following will now type-check:
+`func () : { #tag : Nat8 } = #tag 42;`
+- Warning when the read file has a different name from the requested file.
+- `debug_show` implemented for `Principal`
+- Type hash implementation
```

Comment:
I would say these are all very small/internal changes that can be omitted. We are missing the really BIG BREAKING changes: https://github.com/dfinity/motoko-base/issues/37

Reply:
It's also worth mentioning the open source of `base` repo, and encourage developers to use vessel to download the latest base from master.

Reply:
@lsgunnlsgunn Will we publish the style guide (https://github.com/dfinity-lab/motoko/blob/master/doc/style.md) on the SDK website along with the dfx release? If so, it would be great to promote the guide here.

Reply:
@chenyan-dfinity @lsgunnlsgunn @stanleygjones would it be appropriate to direct folks to that PR now that the repo is open for a summary of the major changes? Vs. listing them out here?

Reply:
Sounds good to me. It's generally good to promote the base repo, and encourage people to check there more often. Note that there are other changes other than this issue. The git logs in the repo is really messy, so it's hard to dig them out....